webapp/src/components/chat/chat-history/ChatHistoryTextContent.tsx

```diff
@@ -19,14 +20,18 @@ interface ChatHistoryTextContentProps {
 export const ChatHistoryTextContent: React.FC<ChatHistoryTextContentProps> = ({ message }) => {
     const classes = useClasses();
 
-    let content = message.content.trim()
-        .replace(/[\u00A0-\u9999<>&]/g, function (i: string) {
-            return `&#${i.charCodeAt(0)};`;
-        });
+    let content = message.content.trim().replace(/[\u00A0-\u9999<>&]/g, function (i: string) {
+        return `&#${i.charCodeAt(0)};`;
+    });
     content = utils.formatChatTextContent(content);
-    content = content
-            .replace(/\n/g, '<br />')
-            .replace(/ {2}/g, '&nbsp;&nbsp;');
+    content = content.replace(/\n/g, '<br />').replace(/ {2}/g, '&nbsp;&nbsp;');
```

Comment:
nit: comments on what these two statements do

Reply:
This was a format autofix so not in the scope of my PR